assets/index.css

```diff
@@ -0,0 +1,1107 @@
+body {
+  margin: 0 auto;
+  max-width: 1310px;
+  color: #333;
+  font-size: 13px;
+  line-height: 18px;
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: #f4f5f7;
+}
+
+svg {
+  fill: white;
+  width: 8rem;
+  cursor: pointer;
+}
+
+header {
+  background-color: #f72210;
+  font-family: 'Roboto Mono', monospace;
+}
+
+.container {
+  max-width: 1310px;
+}
+
+.header-content {
+  padding: 0.7rem 0;
+}
+
+.header-subscribe {
+  display: flex;
+}
+
+.header-user-icon {
+  color: rgba(255, 255, 255, 0.75);
+  font-size: 1rem;
+  padding-right: 0.62rem;
+}
+
+.subscribe-button {
+  font-weight: bolder;
+  background-color: #ffe400;
+  color: black;
+  border: none;
+  border-radius: 0.18rem;
+  padding: 0 0.3rem;
+  height: 1.5rem;
+  font-size: 1rem;
+}
+
+.header-menu {
+  font-size: 1rem;
+  padding: 0 0.8rem;
+  padding-top: 0.29rem;
+  color: rgba(255, 255, 255, 0.55);
+}
+
+.header-login {
+  display: none;
+  color: white;
+  font-weight: bolder;
+  padding-right: 1rem;
+  font-size: 1.1rem;
+}
+
+.date-info {
+  display: none;
+}
+
+.nav-bar {
+  display: none;
+  margin: 0;
+  background-color: white;
+}
+
+.nav-list {
+  border-radius: 0;
+  width: 90%;
+  margin: 1vw 0;
+}
+
+.main-top-story,
+.main-featured-stories,
+.image-turn,
+.image-culture {
+  width: 100%;
+  height: 57.5vw;
+  background-image: url("https://d.newsweek.com/en/full/1596899/pepper-spray.jpg?w=591&h=364&f=e8e41744c32896b5fd2884adfd4497c7");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.image-turn {
+  cursor: pointer;
+  height: 90vw;
+  background-image: url("https://d.newsweek.com/en/full/1598151/spider-man-manhattan-bridge-black-lives-matter.jpg?w=397&h=397&f=e20b37a16dffe1d3fc961d40c4186757");
+}
+
+.main-featured-stories {
+  background-image: url("https://d.newsweek.com/en/full/1597214/derek-chauvin.jpg?w=397&h=265&q=90&f=0f0e6066bb8d29b49e7039b6ced98c57");
+}
+
+.mfs1 {
+  background-image: url("https://d.newsweek.com/en/full/1598644/hong-kong-china-protests-anniversary.jpg?w=397&h=265&q=90&f=1ba7efc031f64fa47bc92ca851d540e3");
+}
+
+.mfs2 {
+  background-image: url("https://d.newsweek.com/en/full/1598498/president-trump-hosts-rolling-remember-ceremony.jpg?w=397&h=265&q=90&f=f349c0b828fa2a62a4538daf277fff49");
+}
+
+.mfs3 {
+  background-image: url("https://d.newsweek.com/en/full/1598604/flynn.jpg?w=397&h=265&q=90&f=da5da73fb23b520dd6e1dcb38e42c128");
+}
+
+.mfs4 {
+  background-image: url("https://d.newsweek.com/en/full/1598654/larry-kudlow.jpg?w=397&h=265&q=90&f=957fc80d3934d9c0248ca8d46a6c02db");
+}
+
+.image-culture {
+  background-image: url("https://d.newsweek.com/en/full/1598154/top-game-1.jpg?w=397&h=397&f=12115b2be0ea94f6da48f2e50f4c6b7a");
+  height: 90vw;
+}
+
+.bottom-tag {
+  background-color: #ffffffe6;
+  width: fit-content;
+  height: fit-content;
+  display: flex;
+  justify-content: center;
+  color: #f72210;
+  font-weight: bolder;
+  align-items: center;
+  padding: 1vw;
```

Comment:
You can use bootstrap classes to display flex and justify-content. Please see this [cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) for more information